pages/common/borg.md

```diff
@@ -0,0 +1,32 @@
+# borg
+
+> Deduplicating backup tool with compression and encryption.
+> Supports local and remote backups, mountable as filesystems.
```

Comment:
Does this mean that the remote backups have to be mounted as a file system, or that backs can be mounted as a file system?

Reply:
It's the backups that can be mountable. But borg can backup remotely over ssh. The command for an init would look like:
$ borg init user@hostname:backup

I wasn't sure whether to include a remote example or not. Wanted to keep it simple :-)

Reply:
Maybe the description could be simplified. It sounds a bit like an ad...